SPECS/mesa-libGLU/mesa-libGLU.spec

```diff
@@ -0,0 +1,130 @@
+Summary:        Mesa libGLU library
+Name:           mesa-libGLU
+Version:        9.0.1
```

Comment:
Thanks for bringing this into `SPECS`! Please also remove the one inside `SPECS-EXTENDED` - it's going to cause issues. I'll add a few changes below to include the clean-up we've already done in that version.

Reply:
Done.